### PR TITLE
[ENH] use full width entity page

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes none

Tiny tweak that helps for all tables but that of MRI.

### Before

image

https://bids-specification.readthedocs.io/en/latest/appendices/entity-table.html#biopotential-amplification-eeg-and-ieeg

### After

image

https://bids-specification--1784.org.readthedocs.build/en/1784/appendices/entity-table.html

### CODE REVIEW

**Comment by effigies** [comment]
Definitely an improvement, but losing navigation doesn't seem great. Maybe just hide the TOC?

Once https://github.com/squidfunk/mkdocs-material/issues/3428 exists, we could see about starting navigation in collapsed mode.

**Comment by Remi-Gau** [comment]
> Definitely an improvement, but losing navigation doesn't seem great. Maybe just hide the TOC?

Let me try.

File: src/appendices/entity-table.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+hide:
+-   navigation
```
Comment (Remi-Gau) [inline src/appendices/entity-table.md:3]:
```suggestion
```

**Review by effigies — APPROVED** [review]
I think this is still an improvement over the old, and it doesn't leave the user stranded, only able to get back to the index.

**Review by tsalo — APPROVED** [review]
I like it!

**Pull request merged**